### DISCUSSION TITLE
Why can't I enter a valid address into TMSL?

### DISCUSSION THREAD

**Discussion by mikec-ncsd** [code4recovery/12-step-meeting-list#673] [Q&A] [answered]
I am attempting to add a new meeting to TMSL. It does not seem to allow me to enter a valid address.
The only way I can enter new addresses is by using a LOCATION that is already defined.

https://ncsandiegoaa.org/meetings
WP v 5.9
TMSL v 3.12.2

![image](https://user-images.githubusercontent.com/62718725/153082746-6b3d4bd2-731b-4727-87ac-51d76a053180.png)

![image](https://user-images.githubusercontent.com/62718725/152882980-581a3181-d862-43f0-9a7d-64d54c2196f0.png)

**Comment by mikec-ncsd** [comment, marked as answer]
I found the Address Geocoding setting set to BETA. I switched it back to legacy and it seems to work. Does this mean the BETA version doesn't work?
![image](https://user-images.githubusercontent.com/62718725/153083381-5a7b6399-2f90-4ccd-a651-bf0d26948847.png)

**Reply by kiyote33** [reply]
Glad you got it working @mikec-ncsd. Yes, it does appear the API Gateway may need some work yet.  Thanks for letting us know about the issue.
